extensions/azure-storage-blob/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,14 +1,16 @@
+---
 name: "Quarkus Support Azure Storage Blob"
-description: "Quarkus Support for Azure Storage Blob"
```

Comment:
Can we keep this "description" part?

Reply:
Good catch. From [QUARKUS EXTENSION METADATA](https://quarkus.io/guides/extension-metadata), the `description` can be provided in the template file. However, in our case, the content of the `description` is exactly/almost the same as `name`, that's why I get them removed.

Reply:
I think Spring Cloud Azure has the same case as quarkus, you can check this [link](https://github.com/Azure/azure-sdk-for-java/blob/5032c95603837463c1647b1af6df2e048f129870/sdk/spring/spring-cloud-azure-autoconfigure/pom.xml#L17).

<img width="476" alt="image" src="https://github.com/quarkiverse/quarkus-azure-services/assets/4465723/6f2c27f1-ade5-43a7-9212-b13f9248086d">

but I'd rather to keep them both.

Reply:
Fair enough. Fixed with commit [add back description](https://github.com/quarkiverse/quarkus-azure-services/pull/116/commits/093a08ec82124847fda541cca242ff0a15209e16).